ding2.install

```diff
@@ -836,3 +836,10 @@ function ding2_update_7072() {
 function ding2_update_7073() {
   ding2_translation_update();
 }
+
+/**
+ * Enable ting_smart_search.
+ */
+function ding2_update_7082() {
+  module_enable(['ting_smart_search']);
```

Comment:
Why use update 82 here? The previous update hook was 73 so this should be 83.

Also `ting_smart_search` requires `opensearch` which is not used by all installs. Please check whether `opensearch` is installed before enabling `ting_smart_search`.

You can check `ding2_update_7048()` for an update hook which uses the same approach.

Reply:
Is fixed

Reply:
Fixed again